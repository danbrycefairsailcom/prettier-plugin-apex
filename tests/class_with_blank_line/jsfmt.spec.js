run_spec(__dirname, ["apex"], { apexBeginClassWithEmptyLine: true });
