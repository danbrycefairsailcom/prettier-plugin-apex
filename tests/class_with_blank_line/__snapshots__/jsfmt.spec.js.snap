// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Format apex: Class.cls: Class.cls 1`] = `
class OuterClass {
  private int x;
  class InnerClass {
    private int y;
  }
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
class OuterClass {

  private int x;
  class InnerClass {

    private int y;
  }
}

`;

exports[`Format apex: Interface.cls: Interface.cls 1`] = `
Interface TestInterface {
  void potato();
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
interface TestInterface {

  void potato();
}

`;
